### PR TITLE
CiscoS300 'show system' template

### PR DESCRIPTION
**Pull request by PavloSkliarenko**
Template + test

### CODE REVIEW

File: ntc_templates/templates/cisco_s300_show_system.textfsm [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Value DESCRIPTION (.*)
+Value UP_TIME (.*)
+Value CONTACT (.*)
+Value HOSTNAME (.*)
+Value List LOCATION (.*)
+Value ADDRESS (.*)
```
Comment (jvanderaa) [inline ntc_templates/templates/cisco_s300_show_system.textfsm:6]:
```suggestion
Value MAC_ADDRESS (.*)
```

**Review by jvanderaa — CHANGES_REQUESTED** [review]

**Comment by PavloSkliarenko** [comment]
@jvanderaa How can I re-run tests, because they failed because of this [PR](https://github.com/networktocode/ntc-templates/pull/1557)?

**Comment by jvanderaa** [comment]
Seeing if my latest re-run will get this.

**Comment by PavloSkliarenko** [comment]
Fix value names according to new Data model

**Pull request merged**